source/jobs/JobsFeature.h

```diff
@@ -77,6 +78,9 @@ namespace Aws
                     virtual int stop() override;
 
                   protected:
+
+                    bool compareJobDocuments(const Aws::Crt::JsonObject& job1, const Aws::Crt::JsonObject& job2);
```

Comment:
Missing API documentation (maybe add some info about how the comparrision is performed).

Reply:
Will be taken care of in the revision.